src/main/kotlin/org/axonframework/intellij/ide/plugin/api/PublishingMethod.kt

```diff
@@ -0,0 +1,32 @@
+package org.axonframework.intellij.ide.plugin.api
+
+import com.intellij.openapi.project.Project
+import com.intellij.psi.JavaPsiFacade
+import com.intellij.psi.PsiMethod
+import com.intellij.psi.search.GlobalSearchScope
+
+enum class PublishingMethod(
```

Comment:
It might be me, but I can't find where this enumeration is used. Would you mind pointing me to the usage of this, @Morlack?